sql.md

```diff
@@ -0,0 +1,100 @@
+#ORDER:
+
+#Find all subjects sorted by subject
+*SELECT subject FROM subjects
+*ORDER BY subjects ASC;
+
+#Find all subjects sorted by location
+*SELECT subject FROM subjects
+*ORDER BY location ASC;
+
+#WHERE
+
+#Find the book Little Women
+*SELECT * FROM books
+*WHERE title = 'Little Women';
+
+#Find all books containing the word Python
+*SELECT * FROM books
+*WHERE title ILIKE '%python%';
+
+#Find all subjects with the location "Main St" sort them by subject
+*SELECT * FROM subjects
+*WHERE location = 'Main St'
+*ORDER BY subject ASC;
+
+#JOINS
+
+#Find all books about Computers list ONLY book title
+*SELECT title FROM books
+*INNER JOIN subjects
+*ON books.subject_id = subjects.id
+*WHERE subject = 'Computers';
```

Comment:
good, here is an alternative way to write it - select books.title from books join subjects on books.subject_id = subjects.id where subjects.subject = 'Computers';